selftests/functional/plugin/test_jsonresult.py

```diff
@@ -0,0 +1,20 @@
+import json
+from os import path
+
+from avocado.utils import process
+from selftests.utils import AVOCADO, TestCaseTmpDir
+
+
+class JsonResultTest(TestCaseTmpDir):
+
```

Comment:
Was this added accidentally?

Reply:
No, it is the test for this feature, so we will be able to catch the #4749 in the future.